CharlieBackend.Data/Repositories/Impl/LessonRepository.cs

```diff
@@ -42,6 +42,33 @@ public async Task<List<Lesson>> GetLessonsForMentorAsync(long? studentGroupId, D
                 .ToListAsync();
         }
 
+        public async Task<List<Lesson>> GetLessonsForStudentAsync(long? studentGroupId, DateTime? startDate, DateTime? finishDate, long studentId)
+        {
+            return await _applicationContext.StudentsOfStudentGroups
```

Comment:
Please write it like this
StudentsOfStudentGroup.Where(
  studentsOfGroup => studentsOfGroup .StudentId == studentId &&
  studentsOfGroup.GroupId == groupId)
.Join(Lessons, studentsOfGroup.StudentGroupId, l=> l.studentGroupId)
Select(LessonDTO)

Reply:
This code works but I cannot retrive `Theme` for the `Lesson`. To retrieve `Theme` we need to add additional `Join` with the `Theme` table. Therefore we need to create new instance of lesson to fill it with all of the lesson data and all of the theme data.
Also, why do we need to map it to the `LessonDTO` if it is the repository level and mapping to dtos objects happens on the service layer.
Thanks)

Reply:
@zelya2021 
I'm sorry, wouldn't adding a simple `Include()` work?
```cs
// ...
.Join(
    _applicationContext.Lessons.Include(lesson => lesson.Theme),
    // ...
```
Looks like it works for me:
![lesson-theme](https://user-images.githubusercontent.com/38729881/108059289-f2a37480-705d-11eb-9f0d-6d980e1b3f75.png)

Reply:
Can you try to do this by .Include() function instead of join? 
Also I don't see using of time arguments. Do we need them?

Reply:
Added a 'Theme' with .Include()